pages/api/ingredient/[mainIngredient].js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export default async function getRecipeByIngredients (req, res) { 
+  const { mainIngredient } = req.query;
```

Comment:
what do you think about getting the ingredient name from a query param like `/ingredient?name=salt`

Reply:
I don't exactly know how to do this in one, I understand how to retrieve this query, but I don't know how to build an API route to get the information like that.